active-rfcs/0000-portals.md

```diff
@@ -0,0 +1,355 @@
+- Start Date: 2020-01-20
+- Target Major Version: (3.x)
+- Reference Issues: (fill in existing related issues, if any)
+- Implementation PR:
+
+# Summary
+
+- Adds a `<portal>` component to Vue core
+- the component accepts a DOM selector via a prop
+- the component moves its children to the element identified by the DOM selector
+- At the virtual DOM level, the children stay descendants of the `<portal>` though, so they i.e. have access to injections from its ancestors
+
+# Basic example
+
+```html
+<body>
+  <div id="app">
+    <h1>Move the #content with the portal component</h1>
+    <portal target="#endofbody">
+      <div id="content">
+        <p>
+          this will be moved to #endofbody.<br />
+          Pretend that it's a modal
+        </p>
+        <Child />
+      </div>
+    </portal>
+  </div>
+  <div id="endofbody"></div>
+  <script>
+    const { Portal } = window.Vue;
+    new Vue({
+      el: "#app",
+      components: {
+        Portal,
+        Child: { template: "<div>Placeholder</div>" }
+      }
+    });
+  </script>
+  <body></body>
```

Comment:
What does this body tag mean?

Reply:
It means I have to fix a typo and remove the superflous opening tag.

That there's a body tag at all is because the example is means to be in HTML, not in an SFC.